docs/resource-policy/policy/balloons.md

```diff
@@ -168,6 +168,11 @@ Balloons policy parameters:
     preferring exclusive CPUs, as long as there are enough free
     CPUs. The default is `false`: prefer filling and inflating
     existing balloons over creating new ones.
+  - `preferIsolCpus`: if `true`, prefer system isolated CPUs for this
```

Comment:
It's a bit inaccurate to say "the workload increases" or "the workload _requires_ more isolated CPUs". A workload can at most wish to be put in a balloon by annotation, and a balloon can only _prefer_ isolcpus. Furthermore, I'd like to provide a warning and guidance in this documentation, because otherwise there are chances that a reader does not understand that here be dragons...

I'd suggest rewording the preferIsolCpus doc:

if `true`, prefer system isolated CPUs (refer to kernel command line parameter "isolcpus") for this balloon. Warning: if there are not enough isolated CPUs in the system for balloons that prefer them, balloons may include normal CPUs, too. This kind of mixed-CPU balloons require special attention when implementing programs that run on them. Therefore it is recommended to limit the number of balloon CPUs (see maxCPUs) and allocate CPUs upfront (see minBalloons, minCPUs) when using preferIsolCpus. The default is `false`.

Reply:
Fixed.